airgapped-rest_api/docker-compose.yml

```
@@ -0,0 +1,47 @@
+version: "3"
+services:
+
+  elasticsearch:
+    image: "docker.elastic.co/elasticsearch/elasticsearch:7.9.2"
+    ports:
+      - 9200:9200
+    restart: on-failure
+    environment:
+      - discovery.type=single-node
+      - "ES_JAVA_OPTS=-Xms1024m -Xmx1024m"
+    healthcheck:
+      test: curl --fail http://localhost:9200/_cat/health || exit 1
+      interval: 10s
+      timeout: 1s
+      retries: 10
+
+  haystack-airgapped:
+    image: haystack_airgapped
+    build:
+      context: ../../haystack-demos
+      args:
+        HAYSTACK_BASE_IMAGE: deepset/haystack:cpu-v1.14.0
+        # In case you want to download, please add the token in the line below.
```

I think this comment refers to private models?
```suggestion
        # In case you want to download private models, please add the HF token in the line below.
```